tests/integration/devfile/cmd_devfile_create_test.go

```diff
@@ -226,7 +226,7 @@ var _ = Describe("odo devfile create command tests", func() {
 		helper.Chdir(context2)
 		helper.CopyExampleDevFile(filepath.Join("source", "devfiles", "nodejs", "devfile.yaml"), filepath.Join(context2, "devfile.yaml"))
 		output := helper.CmdShouldPass("odo", "create", "--starter", "nodejs", "--now")
-		Expect(output).To(ContainSubstring("Changes successfully pushed to component"))
+		Expect(output).To(ContainSubstring("Changeshttps://github.com/odo-devfiles/nodejs-ex.git successfully pushed to component"))
```

Comment:
Should there by a space here? I hope the tests won't fail here. :thinking:

Reply:
it changed by mistake